src/main/java/core/basesyntax/BallNumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class BallNumberSupplier {

```

Comment:
don't need to create a separate class for this logic, move it directly to Lottery

Reply:
Moved this class.